### DISCUSSION TITLE
Committing & merging process clarification

### DISCUSSION THREAD

**Discussion by zlaic** [Zeinab-HasasZiabari/Tafe24S1T102_Vir#7] [Q&A] [answered]
Could you outline the process for committing changes and merging branches?

**Comment by Zeinab-HasasZiabari** [comment, marked as answer]
Sure! Please clone the GitHub repository to your local. 
I am working on "feature_mortgage " to develop the mortgage feature of the calculator. 
Please create the feature branch "feature_currency "  from "development" branch and work on currency feature. 
After you are sure that the codes are working, please commit the changes and push the "feature_currency" branch to the Git Repository. 
Be sure that new branch is created while pushing. ( I suggest reading "Session 04 Class Excersises_230825.docx. specially page 21). 
Then I will push my feature branch to the GitHub Repository. ( we need to take screenshots of the revision graph) 
Each team member will merge the feature branches into the development branch, one at a time.  We will follow the instruction and I will contact you about merging development branch to release branch and finally to the main branch with version tagging. 
We keep contacting at each stage.
